### PR TITLE
feat: nocodb template

### PR DESCRIPTION
**Pull request by DarkPhoenix2704**
Added NocoDB template

### CODE REVIEW

File: templates/nocodb/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      NC_REDIS_URL: ${NC_REDIS_URL}
+    labels:
+      - traefik.enable=true
+      - traefik.http.routers.${HASH}.rule=Host(`${NOCODB_HOST}`)
+      - traefik.http.services.${HASH}.loadbalancer.server.port=${NOCODB_PORT}
+    volumes:
+      - nc_data:/usr/app/data
+
+  root_db:
+    image: postgres
```
Comment (Siumauricio) [inline templates/nocodb/docker-compose.yml]: can you please use a defined version?

File: templates/nocodb/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+version: "3.8"
+services:
+  nocodb:
+    image: nocodb/nocodb:latest
```
Comment (Siumauricio) [inline templates/nocodb/docker-compose.yml]: can you please use a defined version? this in order to prevent break the application in the future

**Review by Siumauricio — CHANGES_REQUESTED** [review]

File: templates/nocodb/docker-compose.yml (hunk 1 above)
Comment (DarkPhoenix2704) [reply templates/nocodb/docker-compose.yml]: updated

File: templates/nocodb/docker-compose.yml (hunk 2 above)
Comment (DarkPhoenix2704) [reply templates/nocodb/docker-compose.yml]: updated

**Review by Siumauricio — APPROVED** [review]
Thank you ❤️ appreciated!

**Pull request merged**